### DISCUSSION TITLE
UX анализ и вопросы

### DISCUSSION THREAD

**Discussion by Darolga** [atls/kino-reporter#10] [General]
Многое верно описано в [Рекомендациях по улучшени юзабилити](https://docs.google.com/document/d/1PXsi0Px1GePh-wXxpOcQTsT7RtfFhjvj78ENQVMYG50/edit#heading=h.b9dtouukwfns), повторяться не буду.

Дополнительно от меня:

**Шапка**

1. Почему выделена рубрика "Подкасты" ярко-красным и плеер вынесен на самое видно мето ? На них как-то зарабатываете?

2. Предлагаю в шапку вынести наиболее популярные рубрики, чаще всего просматриваемые пользователями: новости, кино, сериалы.

3. Иконки соцсетей лучше разместить в подвал. Для шеринга конкретной статить, как описывали сеошники, поместим внутри статьи (плавающие иконки).

4. Поиск нужно заметнее.

5. Кнопку "Подписаться" на новости сейчас совсем не найдешь. А она очень важна для возврата пользователей через анонсы. Надо заметнее, на видном месте. Можно сделать меню фиксированным и кнопка все время будет видна.  Можно через 30-50 минут пребывания на сайте показывать модальное окно с формой подписки.

6. Избранное. Добавлять, просматривать свои избранные, делиться ими после авторизации. 

7. Подписку на журнал лучше переименовать в "Купить журнал". Нравится как подано у https://kinoart.ru/ 
Не спрятано в меню, показывают обложку с кнопкой "Купить" и "Подробнее".
Было бы полезно показать, что внутри журнала, содержание. 

**8. Бесконечная лента на главной без разделения на рубрики.** 
Сейчас все в кучу. Нужно визуально разделить на  рубрики, в которых показываем превью самых свежих (и/или популярных) постов, например:
- Главное сегодня (3-4 превью) + кнопка "Смотреть еще" (с переходом в категорию)
- Самые популярные (3-4 превью) + кнопка "Смотреть еще" (с переходом в категорию) и т.д.
- подборки типо "Топ-5", "Топ-10"

**9. Видео и фото**
Для улучшения поведенческих факторов, а именно для увеличения времени нахождения на сайте, заметнее показывать видео-контент. Тем более ресурс про кино. Видео и фото хорошо разбавляют длинные тексты. 

На главной ленту с трелерами нужно ставить на первом экране. Можно слайдер.

Судя по карте кликов в Яндекс Метрике, много кликают на стрелке перелистывания слайдера Трейлеры. Перенести выше, заметнее, было бы круто запускать трейлер сразу в окне без перехода на Ютюб.

По Яндекс метрике самая популярная статья за год - [25 лучших бесплатных фильмов на Ютюбе](https://kinoreporter.ru/25-luchshih-filmov-kotorye-mozhno-besplatno-posmotret-na-youtube/). Во время новогодных праздников ее посещение резко возрастало. Можно сделать подборку и разместить на главной слайдер с трейлерами, как вариант.
Особенно это актуально сделать сейчас перед новогодними каникулами.

![image](https://user-images.githubusercontent.com/77831845/143565027-88e13925-0f95-44b4-8d6f-ad0275f577c8.png)

Тоже самое касается фото материалов, нужно использовать крупные фото, слайдеры, например, актеров, кадры из фильма - хорошо увеличивают время присутствия на сайте. 

**10. Детальная страница статьи.** 
Бесконечный скролл - нехорошо. В идеале 1 статья = 1 страница, нужна кнока "Наверх".
(Согласна с рекомендациями (https://docs.google.com/document/d/1PXsi0Px1GePh-wXxpOcQTsT7RtfFhjvj78ENQVMYG50/edit#heading=h.b9dtouukwfns)
 
**11. Реклама на сайте**
 Рекламный баннер в левом нижнем углу перекрывает основной контент. Предлагаю заложить плейсхолдеры под рекламу в теле статьи и в сайд-баре. 
Есть ли статистика по наиболее кликабельным блокам?

12. Видео-проигрыватель в правом нижнем углу - реклама или что? 

**13. Вход/регистрация**
Есть корзина, избранные, значит нужен личный кабинет? Посмотреть историю заказов журнала, например. Будем реализовывать?

**14. Каталог** 
Будем ли реализовывать? ([по рекомендациям](https://github.com/torin-asakura/kino-reporter/discussions/2)):
-  календарь кинопремьер (раздел с фильтрами, виджет)
- [база данных фильмов/сериалов/персон](нужно сделать базу данных фильмов/сериалов/персон)
- [Карточка фильма/персоны](https://drive.google.com/file/d/1NsPRw7hcEHayZ0EqlZcMRZ7ZFb2h6byE/view)

**15. Гид по киновселенным**  будет реализован? ([Из дока рекомендаций](https://docs.google.com/document/d/1PXsi0Px1GePh-wXxpOcQTsT7RtfFhjvj78ENQVMYG50/edit#heading=h.63dzxxz6sysb))

16. Аудиоверсии статей (из рекомендаций) - будем реализовывать? 

**15. Логотип и цвет**
Меняем лого или оставляем как есть? 
По цветовой гамме можно оставить черный, красный (как акциент), белый, серый.

**Comment by AlexeyGovorukhin** [comment]
Внимательно изучил рекомендации, со многим согласен - уже были планы и желания реализовать, но с разрабами холдинга это было практически не реализуемо - долго, ненадёжно и плохо.
В целом, чтобы реализовать предложения нужно проработать в целом логику и архитектуру сайта, чтобы он был удобными и интересным для пользователей.

По дополнениям отвечу по тексту:


Шапка

Почему выделена рубрика "Подкасты" ярко-красным и плеер вынесен на самое видно мето ? На них как-то зарабатываете?

**нет, не зарабатываем, была задача продвинуть подкасты, чтобы у них было максимальное количество прослушиваний. Потом страсть к подкастам поутихла, но плеер пока оставили на месте**

Предлагаю в шапку вынести наиболее популярные рубрики, чаще всего просматриваемые пользователями: новости, кино, сериалы.

**у нас получается сложный рубрикатор - есть несколько сущностей: разделы (новости, статьи, рецензии и т.п.) и темы (театр, кино, стиль и т.д.), разделы и темы пересекаются, но смешивать их у нас не получается.**

Иконки соцсетей лучше разместить в подвал. Для шеринга конкретной статить, как описывали сеошники, поместим внутри статьи (плавающие иконки).

**двумя руками "за"**

Поиск нужно заметнее.

**и сам поисковый движок нужен лучше - с пониманием опечаток и пр.**

Кнопку "Подписаться" на новости сейчас совсем не найдешь. А она очень важна для возврата пользователей через анонсы. Надо заметнее, на видном месте. Можно сделать меню фиксированным и кнопка все время будет видна. Можно через 30-50 минут пребывания на сайте показывать модальное окно с формой подписки.

**с подпиской на новости отдельная тема - мы в своё время приостановили после того, как появился весь этот гемор со сбором и обработкой персональных данных. на thr.ru у нас была подписка на новостные рассылки, но там за всё вркемя 100 человек подписалось только, нет никакого смысла**

Избранное. Добавлять, просматривать свои избранные, делиться ими после авторизации.

**То же самое - сбор и обработка персональных данных, юристы запугали, что это проблема**

Подписку на журнал лучше переименовать в "Купить журнал". Нравится как подано у https://kinoart.ru/
Не спрятано в меню, показывают обложку с кнопкой "Купить" и "Подробнее".
Было бы полезно показать, что внутри журнала, содержание.

**тоже тонкий момент - мы не педалируем особо продажу журнала, у нас его ограниченное количество**

8. Бесконечная лента на главной без разделения на рубрики.
Сейчас все в кучу. Нужно визуально разделить на рубрики, в которых показываем превью самых свежих (и/или популярных) постов, например:

Главное сегодня (3-4 превью) + кнопка "Смотреть еще" (с переходом в категорию)
Самые популярные (3-4 превью) + кнопка "Смотреть еще" (с переходом в категорию) и т.д.
подборки типо "Топ-5", "Топ-10"

**согласен, нужно обсудить и придумать логику, важно, чтобы материалы сами подтягивались в рубрики/разделы на главной по определённому принципу**

9. Видео и фото
Для улучшения поведенческих факторов, а именно для увеличения времени нахождения на сайте, заметнее показывать видео-контент. Тем более ресурс про кино. Видео и фото хорошо разбавляют длинные тексты.

На главной ленту с трелерами нужно ставить на первом экране. Можно слайдер.

Судя по карте кликов в Яндекс Метрике, много кликают на стрелке перелистывания слайдера Трейлеры. Перенести выше, заметнее, было бы круто запускать трейлер сразу в окне без перехода на Ютюб.

**согласен, вопрос в реализации, наши не смогли**

По Яндекс метрике самая популярная статья за год - 25 лучших бесплатных фильмов на Ютюбе. Во время новогодных праздников ее посещение резко возрастало. Можно сделать подборку и разместить на главной слайдер с трейлерами, как вариант.
Особенно это актуально сделать сейчас перед новогодними каникулами.

**полагаю, что этот материал лучше не трогать, а сделать новый. Проблема с подобными материалами в том, что со временем некоторые тайтлы блокируют для показа на сторонних сайтах, некоторые удаляют и приходится ставить заглушки.** 

image

Тоже самое касается фото материалов, нужно использовать крупные фото, слайдеры, например, актеров, кадры из фильма - хорошо увеличивают время присутствия на сайте.

**по возможности используем** 

10. Детальная страница статьи.
Бесконечный скролл - нехорошо. В идеале 1 статья = 1 страница, нужна кнока "Наверх".
(Согласна с рекомендациями (https://docs.google.com/document/d/1PXsi0Px1GePh-wXxpOcQTsT7RtfFhjvj78ENQVMYG50/edit#heading=h.b9dtouukwfns)

**сделали бесконечный скролл потому, что ранее после просмотра статьи, на которую приходили из поиска или агрегатора, сразу сваливали обратно, переходов на главную было маловато, поэтому если уходим от бесконечного скролла, считаю крайне важным сделать каталог и насытить материалы гиперссылками со всплывающими сниппетами фильмов/персон, внутренней перелинковкой и т.п.**

11. Реклама на сайте
Рекламный баннер в левом нижнем углу перекрывает основной контент. Предлагаю заложить плейсхолдеры под рекламу в теле статьи и в сайд-баре.
Есть ли статистика по наиболее кликабельным блокам?

**левый нижний - это mail.ru, поставили под трафикообмен. статистики особой нет, но точно понимаем, что лучше всего первый-второй экран и внутренние первый экран. сильно перегружать рекламой не хотим.
https://www.figma.com/file/PmFmyixd4NNOIxoDcuH2qGhl/%D0%9A%D0%B8%D0%BD%D0%BE%D1%80%D0%B5%D0%BF%D0%BE%D1%80%D1%82%D0%B5%D1%80-%D0%B2%D1%91%D1%80%D1%81%D1%82%D0%BA%D0%B0-%D0%B4%D1%80%D0%B0%D1%84%D1%82
https://www.figma.com/file/PmFmyixd4NNOIxoDcuH2qGhl/%D0%9A%D0%B8%D0%BD%D0%BE%D1%80%D0%B5%D0%BF%D0%BE%D1%80%D1%82%D0%B5%D1%80-%D0%B2%D1%91%D1%80%D1%81%D1%82%D0%BA%D0%B0-%D0%B4%D1%80%D0%B0%D1%84%D1%82?node-id=10%3A0**

Видео-проигрыватель в правом нижнем углу - реклама или что?
**реклама**

13. Вход/регистрация
Есть корзина, избранные, значит нужен личный кабинет? Посмотреть историю заказов журнала, например. Будем реализовывать?

**сбор и обработка персональных данных. личный кабинет не особо то и нужен. а вот реализацию магазина точно нужно улучшить**

14. Каталог
Будем ли реализовывать? (по рекомендациям):

**календарь кинопремьер (раздел с фильтрами, виджет) кинопремьеры под вопросом -чаще заходят на специализированные ресурсы (кинопоиск, афиша, кинометро), где есть подробные кинокалендари**
[база данных фильмов/сериалов/персон](нужно сделать базу данных фильмов/сериалов/персон)
**да. в первую очередь будем делать по тем фильмам/сериалам/персонам, материалы про которые уже есть на сайте (соответственно, на карточку после описания подтягиваются релевантные материалы, а в материалах делаем гиперссылку на карточку) +  грядущие громкие премьеры**
Карточка фильма/персоны
есть понимание как ходим https://www.figma.com/file/W8jFgnvrx5AMrY76sx7x2h/%D0%9A%D0%B0%D1%80%D1%82%D0%BE%D1%87%D0%BA%D0%B0-%D1%84%D0%B8%D0%BB%D1%8C%D0%BC%D0%B0?node-id=0%3A1
**лендинг для всех каталогов единый, но по дизайну как его, так и подразделов фильмы/сериалы/персоны нужно думать
https://www.figma.com/file/PmFmyixd4NNOIxoDcuH2qGhl/%D0%9A%D0%B8%D0%BD%D0%BE%D1%80%D0%B5%D0%BF%D0%BE%D1%80%D1%82%D0%B5%D1%80-%D0%B2%D1%91%D1%80%D1%81%D1%82%D0%BA%D0%B0-%D0%B4%D1%80%D0%B0%D1%84%D1%82?node-id=408%3A0**

15. Гид по киновселенным будет реализован? (Из дока рекомендаций)

**подобных гидов несчётное множество, есть ли шанс попасть в топ с ещё одним?**

Аудиоверсии статей (из рекомендаций) - будем реализовывать?
**да, давно думаем об этом! нужны рекомендации по движкам**

15. Логотип и цвет
Меняем лого или оставляем как есть? 
По цветовой гамме можно оставить черный, красный (как акциент), белый, серый.

**в принципе, логотип можно в любой цвет, по гайдлайнам нет ограничений, но зачем?**

**Comment by Darolga** [comment]
@AlexeyGovorukhin 
Предварительная схема структуры сайта и Главной: [https://whimsical.com/CYtZknZeGC237GKYWvEtMQ](https://whimsical.com/CYtZknZeGC237GKYWvEtMQ)
Прямо там онлайн можно редактировать. 
Если редактировать не получится, напишите мне свою почту, вышлю инвайт.
